Cargo.toml

```diff
@@ -24,6 +24,15 @@ include = [
 license = "MIT OR Apache-2.0"
 readme = "README.md"
 
+[workspace.dependencies]
+clap = { path = ".", version = "3.1.10", default-features = false, features = ["std"] }
+clap_derive = { path = "./clap_derive", version = "=3.1.18" }
+clap_lex = { path = "./clap_lex", version = "0.2.0" }
```

Comment:
I think I'd rather not consolidate the workspace members as I expect this will make the release process more difficult.  Without it, I can choose not to release a dependent of `clap` but with it, I'll have to release all dependents of `clap`.

Also `clap_derive` is not a public dependency for anything besides `clap` to depend on and the use of `=` is specific to clap.

Reply:
I'll make sure to undo this. It looked like an easy way to show off many of the features of workspace inheritance in one place